### ISSUE TITLE
Subtítulos.es

### ISSUE THREAD

**Issue by adirgan** [patrickdessalle/periscope#45]
Modify the plug Subtitulos.es to TuSubtitulo.com since the previous stopped working

**Comment by alexandregz** [comment]
I make the tusubtitulo.com plugin.

You can use this branch while waiting for @patrickdessalle :smile: 
 https://github.com/alexandregz/periscope/commit/8cbead2a1d34cdcf46ed22dc6a297c4e868684a1

**Comment by adirgan** [comment]
I did not work , find the subtitle but not download , and only with this plugin try to force nothing

**Comment by alexandregz** [comment]
Yeah, I know, I deleted because site crew from tusubtitulo doesn't wants scrappers or external services to use the web

**Comment by davibe** [comment]
how do we workaround this ?
